cmd/operator-sdk/up/local.go

```diff
@@ -57,6 +57,7 @@ kubernetes cluster using a kubeconfig file.
 	upLocalCmd.Flags().StringVar(&operatorFlags, "operator-flags", "", "The flags that the operator needs. Example: \"--flag1 value1 --flag2=value2\"")
 	upLocalCmd.Flags().StringVar(&namespace, "namespace", "", "The namespace where the operator watches for changes.")
 	upLocalCmd.Flags().StringVar(&ldFlags, "go-ldflags", "", "Set Go linker options")
+	upLocalCmd.Flags().BoolVar(&enableRaceDetector, "enable-race-detector", false, "Enables the go runtime's race detector when compiling the operator")
```

Comment:
WDYT about use the same nomenclature? I mean --race only?

Reply:
I don’t mind changing it 🙂. I didn’t do it initially because the other options don’t exactly match the compiler e.g ‘go-ldflags’. But I like and appreciate your suggestion because I am horrible with variable names and this makes it easier.